CascadeEnergy/dynamoDb-marshaler
# rewrote using es5 compatible to node 0.10

Only things which could be considered breaking changes are:

1. I removed any support for passing in an es6 Set. The feature wasn't documented in the README, not sure we ever used it at Cascade, and it kind of made no sense anyway. All it did was convert a set to an array anyway. `Array.from(mySet)`. So if something breaks, just do that conversion before you send it into the marshaler.

2. There used to be an unminified `dist/dynamodb-marshaler.js`. Now there's only the minified version.

The interface of the module doesn't change, but if someone out there was using es6 sets and sending them into the marshaler...it will explode. so I think that makes this a `npm version major` before you `npm publish`.

**To Test**

```
npm test
npm run lint
npm run cov
npm run build:browser
node example/example-marshal.js
node example/example-unmarshal.js
```

And if you open up `example/example.html` in a browser that should work.
